templates/endpoints_v2/examples/CustomModels.ipynb

```diff
@@ -88,6 +88,27 @@
     "```\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We have provided a starter yaml file and you can edit the model to the specific model file that you created.\n",
```

Comment:
looks good for now but we should really consider having another entrypoint that can generate some config files

Reply:
Agree. @shrekris-anyscale and I have filed a jira issue. Who should be the best person to work on this and also have time?